version/version.go

```diff
@@ -6,7 +6,7 @@ var (
 
 const (
 	OperatorName      = "managed-velero-operator"
-	VeleroImageTag    = "velero@sha256:a60096c63ed34621a3d6fc69a02a25b1e1edb4396af891de98b0bcc91120231e"                // quay.io/konveyor/velero:oadp-1.2-amd64
-	VeleroAwsImageTag = "velero-plugin-for-aws@sha256:a9259c6fb71a7ac7d50845cf3d79dfc683700156ab8011c40d5f092c85818f64" // quay.io/konveyor/velero-plugin-for-aws:oadp-1.2-amd64
-	VeleroGcpImageTag = "velero-plugin-for-gcp@sha256:85a9d667ce44855bd1955ba1438c394e44c926d827520914f160d288f48a2525" // quay.io/konveyor/velero-plugin-for-gcp:oadp-1.2-amd64
+	VeleroImageTag    = "oadp-velero-rhel8@sha256:bdf6c3b957c8e4b02d83559c03a4cbbb08fd893eed6f3b04f9164cdddf9c3868"                // registry.redhat.io/oadp/oadp-velero-rhel8:1.2.0-37
```

Comment:
```suggestion
	VeleroImageTag    = "oadp-velero-rhel8@sha256:bdf6c3b957c8e4b02d83559c03a4cbbb08fd893eed6f3b04f9164cdddf9c3868"                // registry.redhat.io/oadp/oadp-velero-rhel8:v1.2
```

This tag would require less frequent changes.. assuming you would want a floating tag.. otherwise LGTM!

Reply:
Perhaps a daily periodics that pull in v1.2 tags and if it changes, fail it so you know to bump the hashes here either manually or programmatically

Reply:
The comment is more to know when the last time we picked an updated tag was. I think we're fine using this for now, at least until MVO is fully deprecated it. If there's some show stopping bug we need to bump, we can bump it here and update both the digest and the comment with the new version